README.md

```diff
@@ -357,9 +366,18 @@ void GREENTEA_SETUP(const int timeout, const char *host_test_name) {
 	while (1) {
```

Comment:
You added the `runTests = true` bit down below, but I think you'll want to include the `!runTests` condition in this while loop (in addition to the variable declaration itself).

Reply:
That is correct. Will fix.